dfusion_rust_core/src/models/standing_order.rs

```diff
@@ -48,54 +63,133 @@ impl From<mongodb::ordered::OrderedDocument> for StandingOrder {
     fn from(document: mongodb::ordered::OrderedDocument) -> Self {
         let account_id = document.get_i32("_id").unwrap() as u16;
         let batch_index = U256::from(document.get_i32("batchIndex").unwrap());
+        let valid_from_auction_id = U256::from(document.get_i32("validFromAuctionId").unwrap());
```

Comment:
This field doesn't seem to be present. I assume you have to add it to the group clause here: https://github.com/gnosis/dex-services/blob/master/driver/src/db_interface.rs#L123